src/docs/reference/modules/Flow/BagPart.md

```diff
@@ -17,6 +17,40 @@ An example of this can be found in TheAgencyTheme where four Named BagParts are
 - About
 - Team
 
+## Blocks Editor
+
+The blocks editor provides an alternative editor for BagPart that uses a modal-based content type picker instead of the standard dropdown menu. Content types displayed in the picker can be organised with categories and thumbnails. See [Content Type Settings for Block Pickers](../ContentTypes/README.md#content-type-settings-for-block-pickers) for configuration details.
+
+### Enabling the Blocks Editor
+
+To enable the blocks editor for a BagPart, set the **Editor** option to `Blocks` in the content type definition.
```

Comment:
Add a note that it has to be configured for backward compatibility. One day this could be the default.